test/lib/utils.py

```diff
@@ -37,3 +39,33 @@ def write_test_environment(test_env_file:str):
     # Write tool version info
     with open(test_env_file, "w") as file:
         file.write(markdown_content)
+
+# def insert_at_front(lst, item):
+#     """
+#     Inserts an item at the front of the list.
+    
+#     :param lst: List to insert the item into
+#     :param item: Item to insert
+#     :return: Modified list with the item inserted at the front
+#     """
+#     lst.insert(0, item)
+#     return lst
+
+# def remove_read_only(path):
+#     # Remove read-only attribute on a file or directory
+#     def onerror(func, path, exc_info):
+#         # Is the error an access error?
+#         if not os.access(path, os.W_OK):
+#             os.chmod(path, stat.S_IWUSR)
+#             func(path)
+#         else:
+#             raise
+
+#     if os.path.isdir(path):
+#         for root, dirs, files in os.walk(path):
+#             for dir in dirs:
+#                 os.chmod(os.path.join(root, dir), stat.S_IWUSR | stat.S_IREAD)
+#             for file in files:
+#                 os.chmod(os.path.join(root, file), stat.S_IWUSR | stat.S_IREAD)
+#     else:
+#         os.chmod(path, stat.S_IWUSR | stat.S_IREAD)

```

Comment:
Is there a reason for this commented-out code or is it a left-over?

Reply:
This commented coed is redundant. Removed it in [Addressed review comments](https://github.com/Open-CMSIS-Pack/cmsis-toolbox/pull/155/commits/3457137cda2e15eaf7d669e1cbd37dcd96b63dbc)